dcpy/connectors/socrata/publish.py

```diff
@@ -329,7 +329,8 @@ def upload_attachment(
         content_types = {
             ".csv": "text/csv",
             ".pdf": "application/pdf",
-            ".xlsx": "application/vnd.ms-excel",
+            ".xlsx": "application/octet-stream",
+            ".xls": "application/octet-stream",
```

Comment:
Are we pushing any `xls`s??

Reply:
not yet - but that was the only other format I could envision, other than various textfile formats